File: infinite-arpg/Scenes/CoreGameScene.cs
using System;
using Microsoft.Xna.Framework.Graphics;
using Microsoft.Xna.Framework;
using IronPython.Hosting;
using Microsoft.Scripting;
using Microsoft.Scripting.Hosting;

namespace infinitearpg
{
    public class CoreGameScene : AbstractScene
    {
        private readonly Texture2D player;
        private readonly ScriptEngine pythonEngine = Python.CreateEngine();

        public CoreGameScene(GraphicsDevice graphicsDevice) : base(graphicsDevice)
        {
            this.player = this.LoadImage("Content/player.png");
            var source = this.pythonEngine.CreateScriptSourceFromString("print('HI!!!!!')");
            source.Execute();
        }

        public override void Draw(SpriteBatch spriteBatch)
        {
            spriteBatch.Draw(player, new Vector2(300, 100));
        }
    }
}

File: infinite-arpg/Scripts/CoreGameScene.py
from Microsoft.Xna.Framework import Vector2

from DeenGames.InfiniteArpg.Scenes import AbstractScene

class CoreGameScene(AbstractScene):
    def __init__(self, graphicsDevice):
        super(CoreGameScene, self).__init__()
        self.player = self.LoadImage("Content/player.png")

    def Draw(self, spriteBatch):
        spriteBatch.Draw(self.player, Vector2.Zero)